### ISSUE TITLE
Remove `MCMC_walkthrough.broken`?

### ISSUE THREAD

**Issue by abhisrkckl** [nanograv/PINT#1676]
IMO, there is no point in having a broken example. If needed, we can easily recover this from the git history.

**Comment by scottransom** [comment]
Yeah.  That's bizarre.  Especially since `MCMC_walkthrough.broken` is nearly identical to `MCMC_walkthrough.md`...
